compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -958,6 +958,11 @@ OMR::SymbolReferenceTable::methodSymRefFromName(TR::ResolvedMethodSymbol * ownin
    //
 
    TR_OpaqueMethodBlock *method = fe()->getMethodFromName(className, methodName, methodSignature);
+
+   // It is possible for getMethodFromName to return NULL in relocatable compilations
+   if (!method && comp()->compileRelocatableCode())
+      comp()->failCompilation<TR::CompilationException>("Failed to get method from name\n");
```

Comment:
Note sure if `failCompilation` supports it but would be useful when this happens to have the `className` and `methodName` in the failure string.

Reply:
`failCompilation` doesn't support that; I regret not adding that capability when I first added `failCompilation` but it seems more and more something that should be there.

Reply:
I'm working on a changeset for this; I'll open a PR when I've tested that it works fine.